notebooks/sprints/sprint19/530_rsclient_staging.ipynb

```diff
@@ -0,0 +1,520 @@
+    "if os.getenv(\"RSPY_LOCAL_MODE\") == \"1\":\n",
+    "    href = \"http://rs-server-cadip:8000\"\n",
+    "    href_adgs = \"http://rs-server-adgs:8000\"\n",
+    "    href_staging = \"http://rs-server-staging:8000\"\n",
+    "else:\n",
+    "    href = href_adgs = os.environ[\"RSPY_WEBSITE\"]\n",
+    "    href_staging = \"https://rsserverstaging.dev-rspy.esa-copernicus.eu\"\n",
+    "    session.cookies.set (\"session\", os.environ[\"RSPY_OAUTH2_COOKIE\"])\n",
+    "\n",
+    "cadip_collection_id = \"cadip_sentinel1\" #\"cadip_s1A\"\n",
+    "adgs_collection_id = \"adgs\" #\"aux_interval\"\n",
+    "TIMEOUT = 10"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "b752d13b-0316-4f00-9d67-5b702d7b8bdf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a test collection \n",
+    "collection = create_test_collection()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f2ce641b-98ba-4221-b4e2-12cdd54190b9",
+   "metadata": {},
+   "source": [
+    "### Check the newly created collection with the rs-server-catalog. It should be empty."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "1d3f88cc-d31b-4342-ac68-4372b6db1362",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Number of items found in the 'my_test_collection' collection: 0\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Check the catalog for agrosu_my_test_collection\n",
+    "result = session.get(f\"{stac_client.href_catalog}/catalog/collections/{user}:{TEST_COLLECTION}/items\")\n",
+    "catalog_collection = result.json()\n",
+    "assert catalog_collection.get(\"type\") == \"FeatureCollection\"\n",
+    "print(f\"Number of items found in the '{TEST_COLLECTION}' collection: {catalog_collection.get('context').get('returned')}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f989069a-4d84-450f-99a0-1b0b5f3b9292",
+   "metadata": {},
+   "source": [
+    "### Getting all the sessions from the cadip_s1A collection found in the configuration of CADIP station"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "4a176170-7211-4c42-b3de-305d8782874c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Session S1A_20231120061537234567 has 60 assets with datetime 2023-11-20T06:15:37.234000+00:00\n",
+      "Session S1A_20220715090550123456 has 40 assets with datetime 2022-07-15T09:05:50.123000+00:00\n",
+      "Session S1A_20210410031928012345 has 21 assets with datetime 2021-04-10T03:39:28.012000+00:00\n",
+      "Session S1A_20200105072204051312 has 60 assets with datetime 2020-01-05T18:52:26.165000+00:00\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = session.get(f\"{href}/cadip/collections/{cadip_collection_id}/items\")\n",
```

Comment:
@vprivat-ads please note that we do not have a wrapped cadip_client function for these STAC requests, because we need to update CadipClient so it inherits pystac_client.Client, as StacClient does (that should be renamed CatalogClient to avoid confusion, I think). Do we have a Jira for this ?